### PR TITLE
Project-5 Anna Almestål

### PR DESCRIPTION
**Pull request by anna-ip**
This week have been really hard, until mid-day Thursday I had no clue of what I was doing, Thursday noon I met up with some of my class-mates and with some help I managed to get it all together by the end of the day and I think I do understand some of what I've done this week. 

Netlify link [https://cluedo-annaalmestal.netlify.com/](url)

### CODE REVIEW

**Review by ericaglimsholt — CHANGES_REQUESTED** [review]
Hi Anna, your Netlify link doesn't work. Can you take a look at it and add a working one here?

**Comment by anna-ip** [comment]
Hej Erica

Här kommer den https://cluedo-annaalmestal.netlify.com <https://cluedo-annaalmestal.netlify.com/>

//Anna

File: code/clue.js [hunk 1]
```diff
@@ -10,55 +10,241 @@ const mrGreen = {
...
+  age: 63,
+  image: "assets/mustard.png",
+  occupation: "Military"
+}
+
+const mrsWhite = {
+  firstName: "Mrs",
+  lastName: "White",
+  color: "Pink",
+  description: "She....",
```
Comment (ericaglimsholt) [inline code/clue.js:57]: Interesting descriptions of the persons!

**Review by ericaglimsholt — APPROVED** [review]
Good work on your project! One thing to think about is to make to code more easier to read. Comments helps, but there's a lot of out commented code (like functions) that doesn't need to be there at all. It's a good thing to start working on readable code even now in the start. That's something people look like when they're hiring a new developer! Good luck!

**Pull request closed without merging**